API.md

```diff
@@ -8,8 +8,8 @@
 
 ### Properties
 
-- [#app](API.md##app)
```

Comment:
I assume this file is autogenerated. Am I right? So this error will probably popup again.